src/test/scala/io/qbeast/context/QbeastConfigTest.scala

```diff
@@ -25,16 +25,16 @@ class QbeastConfigTest extends AnyFlatSpec with Matchers with QbeastIntegrationT
 
   "Qbeast config" should "use default configurations" in withSpark { _ =>
     config.DEFAULT_CUBE_SIZE shouldBe 5000000
-    config.CUBE_WEIGHTS_BUFFER_CAPACITY shouldBe 100000
+    config.CUBE_DOMAINS_BUFFER_CAPACITY shouldBe 100000
   }
 
   it should "change configurations accordingly" in withExtendedSpark(
     sparkConfWithSqlAndCatalog
       .set("spark.qbeast.index.defaultCubeSize", "1000")
-      .set("spark.qbeast.index.cubeWeightsBufferCapacity", "1000")
+      .set("spark.qbeast.index.cubeDomainsBufferCapacity", "1000")
       .set("spark.qbeast.index.numberOfRetries", "10")) { _ =>
     config.DEFAULT_CUBE_SIZE shouldBe 1000
-    config.CUBE_WEIGHTS_BUFFER_CAPACITY shouldBe 1000
+    config.CUBE_DOMAINS_BUFFER_CAPACITY shouldBe 1000
```

Comment:
why are we changing this param here?

Reply:
(nothing against it, just to know if has something to do with staging data manager or not)

Reply:
It has nothing to do with the Staging Data Manager.

Reply:
In general, we should not solve different things in the same PR, especially if config variables are involved. They seem harmless unless someone uses this configuration somewhere... and breaks without notice.

If you are confident about the safeness of the new usage, it's fine for me.